rosplan_knowledge_msgs/msg/DomainFormula.msg

```diff
@@ -1,4 +1,5 @@
 # A knowledge item used to represent an atomic formula from the domain.
 # typed_parameters matches label -> type
 string name
+string[] labels
```

Comment:
I'm not sure about this. DomainFormula is used to store the unrounded predicate, meaning that typed_parameters will store: Label->Type. I get that you are using it here to store the grounded proposition, and so you need to store instead: Label->Object; however, you could do this in the existing KeyValue array. (The type is not needed).